traduki/sqla.py

```diff
@@ -120,37 +122,33 @@ def _do_compare(self, op, other, escape):
             ]
             return self.has(or_(*ops))
 
-    class TranslationExtension(AttributeExtension):
-        """AttributeExtension to override the behavior of .set, to accept a dict as new value."""
+    def translation_attribute_set_hook(state, value, oldvalue, initiator):
+        """Set accessor for the `Translation` object.
 
-        @staticmethod
-        def set(state, value, oldvalue, initiator):
-            """Set accessor for the `Translation` object.
+        :note: The value is copied using dict to avoid 2 objects
+            referring to the same Translation. Also the oldvalue should
+            wipe the values for the languages that are not in the value.
 
-            :note: The value is copied using dict to avoid 2 objects
-                referring to the same Translation. Also the oldvalue should
-                wipe the values for the languages that are not in the value.
+        :param state: SQLAlchemy instance state.
+        :param value: The value that is being assigned.
+        :param oldvalue: The current value.
+        :param initiator: SQLAlchemy initiator (accessor).
+        """
+        if value is None or value in (NO_VALUE, NEVER_SET):
```

Comment:
For future compatibility, maybe check `value` against supported types instead?

Reply:
done